docs/Validation Rules.md

```diff
@@ -1,17 +1,15 @@
-# Implemented validation rules
```

Comment:
The underline styles are all Markdown, not mixed.

They do not mention it in the [GitHub Markdown Basics](https://help.github.com/articles/markdown-basics/), but they do in the 
[official Markdown docs basics](https://daringfireball.net/projects/markdown/basics) and [detailed](https://daringfireball.net/projects/markdown/syntax#header), and in this [handy GitHub-centric cheat sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers).

You can see it renders correctly [here](https://github.com/scottclowe/pykwalify/blob/improve-docs/docs/Validation%20Rules.md).

If you prefer the atx-style headings with all `#`, I can swap them to that. The motivation was it made it a lot easier for me to tell the depth of the content when editing the unparsed .md file. I find the difference between `-----------------` and `###` is much more striking between `##` and `###`. But I am happy to change them back for internal consistency if you prefer that.